azurerm/internal/services/netapp/data_source_netapp_volume.go

```diff
@@ -61,6 +62,18 @@ func dataSourceArmNetAppVolume() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+
+			"protocols": {
+				Type:     schema.TypeList,
+				Computed: true,
+				MaxItems: 2,
+				Elem: &schema.Schema{Type: schema.TypeString,
+					ValidateFunc: validation.StringInSlice([]string{
```

Comment:
We don't need validation on a data source

Reply:
Fixed.